### PR TITLE
Video player component

### PR DESCRIPTION
**Pull request by monkemedia**

### CODE REVIEW

File: components/media/VideoPlayer.stories.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { storiesOf } from '@storybook/vue';
+
+import VideoPlayer from './VideoPlayer.vue';
+
+storiesOf('Media', module)
```
Comment (rwd) [inline components/media/VideoPlayer.stories.js]: Let's get some nesting going here, by replacing this with:
```js
storiesOf('Media/Video Player', module)
```

File: components/media/VideoPlayer.stories.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  .add('Video Player', () => ({
```
Comment (rwd) [inline components/media/VideoPlayer.stories.js]: Let's make the "story" more indicative of a specific use of the component:
```js
.add('WebM video', () => ({
```
... and also add a second story with an H.264 MP4, using the URL: https://europeana1914-1918.s3.amazonaws.com/attachments/236696/9555.236696.original.mp4

File: components/media/VideoPlayer.vue [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    >
+  </video>
+</template>
+
+<script>
+  export default {
+    name: 'VideoPlayer',
+
+    props: {
+      url: {
```
Comment (rwd) [inline components/media/VideoPlayer.vue]: Give all the other props have the same name as the attribute they feed, let's call this one `src`.

File: pages/record/_.vue [hunk 4]
```diff
@@ -67,7 +68,8 @@
       AlertMessage,
       WebResources,
       MetadataField,
-      MediaImage
+      MediaImage,
+      VideoPlayer
```
[the remark this answers is not in the document]
Comment (monkemedia) [reply pages/record/_.vue]: This should have been removed. My bad

File: components/media/VideoPlayer.vue [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        type: Boolean,
+        default: false
+      }
+    }
+  };
+</script>
+
+<style lang="scss" scoped>
+
+</style>
```
Comment (mirjamverloop) [inline components/media/VideoPlayer.vue]: To keep it clean, let's remove this empty <style>.

**Pull request merged**